README.md

```diff
@@ -183,17 +183,17 @@ Passing options to the pass can be done using either an array or the
 SpanOptions wrapper object. The following keys are valid:
 
 - `start_time` is a float, int or `\DateTime` representing a timestamp with arbitrary precision.
-- `child_of` is an object of type `OpenTracing\SpanContext` or `OpenTracing\Span`.
-- `references` is an array of `OpenTracing\SpanReference`. 
+- `references` is an array of `OpenTracing\Reference`. 
 - `tags` is an array with string keys and scalar values that represent OpenTracing tags.
 
 ```php
 <?php
 
+use OpenTracing\Reference;
 use OpenTracing\SpanOptions;
 
 $span = $tracer->createSpan('my_span', SpanOptions::create([
-    'child_of' => $parentContext,
+    'references' => Reference::create(Reference::CHILD_OF, $spanContext),
```

Comment:
It makes sense, I removed because we agreed on not to make other references 2nd class citizens but I just checked the API in other languages and yes, makes sense to include it.